config/config.exs

```diff
@@ -1,3 +1,7 @@
 import Config
 
 config :phoenix, :json_library, Jason
+
+config :nimble_template, hex_package_resource: NimbleTemplate.Hex.Package
+
+import_config "#{config_env()}.exs"
```

Comment:
How about adding a check if the config exists or not? so we don't need to create a blank configuration file for dev and prod.
```suggestion
if File.exists?("config/#{config_env()}.exs") do
  import_config "#{config_env()}.exs"
end
```

Reply:
updated in https://github.com/nimblehq/elixir-templates/pull/284/commits/0a89e641f577bdf2e4e0b1545e3dea1f7567369f